conf/mesos-site.xml

```diff
@@ -99,12 +109,12 @@
 
   <property>
     <name>mesos.hdfs.journalnode.cpus</name>
-    <value>1</value>
+    <value>0.5</value>
```

Comment:
SGTM. Just want to make sure we're consistent in the xml defaults and the defaults hardcoded into SchedulerConf.java (all the more reason I want to remove or comment out the xml values and leave the defaults to SchedulerConf.java)